framework/Source/Operations/GaussianBlur.swift

```diff
@@ -20,16 +20,16 @@ public class GaussianBlur: BasicOperation {
         self.useMetalPerformanceShaders = true
         
         ({blurRadiusInPixels = 2.0})()
-        
         if #available(iOS 9, macOS 10.13, *) {
-            self.metalPerformanceShaderPathway = usingMPSImageGaussianBlur
```

Comment:
Suggestion: If we want to keep the cleaner method in the upper level, we can instead just make it static
```suggestion
            self.metalPerformanceShaderPathway = Self.usingMPSImageGaussianBlur
```

Reply:
No, unfortunately, we can't, the function invokes an instance property inside itself, so we can't just make it static

Reply:
Oops you're right I didn't see it using instance properties.